po/zh_CN.po

```diff
@@ -8501,16 +8501,16 @@ msgstr "重载"
 #: pkg/storaged/lvm2/physical-volume.jsx:119
 #: pkg/storaged/stratis/filesystem.jsx:226 pkg/storaged/stratis/pool.jsx:518
 msgid "Usage"
-msgstr "用法"
+msgstr "负载状态"
 
 #: pkg/storaged/storage-controls.jsx:206
 msgid "Usage of $0"
-msgstr "$0 的用法"
+msgstr "$0 的占用率"
```

Comment:
This changes "usage" to "occupancy" in the English translation. This seems at least plausible, but not really consistent with the other translations?